client/components/domains/connect-domain-step/connect-domain-step-verification-status-parsing.js

```diff
@@ -60,9 +59,7 @@ export function getMappingVerificationErrorMessage( mode, verificationStatus ) {
 		return createInterpolateElement(
 			sprintf(
 				/* translators: %s: the list of IP addresses. (Ex.: "192.168.0.1, 192.168.0.2") */
-				__(
-					'The name A records for your domain are set to: <em>%s</em>. Please try this step again.'
-				),
+				__( 'The A records for your domain are set to: <em>%s</em>. Please try this step again.' ),
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.